### PR TITLE
26436: Update form for legislative changes effective 8/1

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As an EDU team member, I need digital Form 22-5490 on VA.gov to be updated to include legislative changes that effective 8/1 so that the form is compliant with the changes.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/26436

## Testing done
Testing passes locally

## Screenshots
Screen Shot 2021-07-13 at 12 06 03 PM
Screen Shot 2021-07-13 at 12 22 13 PM
Screen Shot 2021-07-13 at 4 31 16 PM

## Acceptance criteria
- [x]  On chapter 1, page 1, when the applicant has answers the "Your date of birth" question with a birthdate that reveals the applicant is under the age of 18 the following content is conditionally displayed using the blue information box pattern:

    > Applicants under the age of 18 cannot legally make a benefits election.  Based on your date of birth, please have a parent, guardian or custodian review the information on this application and click the “Submit application” button at the end of this form.   

    a. The content has: aria-live=“polite”

- [x] On chapter 1, page 1, the question "What’s your relationship to the service member whose benefit is being transferred to you?" answer "Spouse or surviving spouse" has a "Learn more" link that:
    a. Opens the following page in a new tab: http://www.va.gov/opa/marriage/
    b. Has the aria-label: "Learn more about VA requirements for marriage certification"

- [x] In chapter 2, page 1, the "Select the benefit that is the best match for you:" question answer has a `aria-describedby="[Informational Content UNIQUE ID]"`
    a. The 'Informational Content Unique Id' corresponds to the conditional content that is displayed after selecting an answer.

- [x] In chapter 2, page 2, question "Before this application, have you ever applied for or received any of the following VA benefits?":
    a. Does not have a "None" option as an answer choice.
    b. The "Other benefit" option has a checkbox
    c. The free text input corresponding with the "Other benefit" option is hidden and content is cleared when the checkbox (AC4b) is unchecked.
    d. The free text input corresponding with the "Other benefit" option is displayed when the checkbox (AC4b) is checked.
    e. When The "Other benefit" option is checked; the question, all available answers, and conditional free text input are displayed with the blue bar pattern.

- [x]  In chapter 6, page 1, the question "Type of education or training" is not required.
- [x]  In chapter 8, page 1, the following I Certify content is displayed after the section accordions:
    a. If the applicant answered the "Your date of birth" with a birthdate that shows that they are under 18 years of age and answered "Have you ever served on active duty in the armed services?" with "No":

    > **By submitting this form** you certify that:
    > 
    > - You are the parent, guardian, or custodian of the applicant
    > 
    > - All statements in this application are true and correct to the best of your knowledge and belief

    b. If the applicant answered the "Your date of birth" with a birthdate that shows that they are under 18 years of age and answered "Have you ever served on active duty in the armed services?" with "Yes" and **_any_** service period "Service end date" is blank:

    > **By submitting this form** you certify that:
    > 
    > - You are the parent, guardian, or custodian of the applicant
    > 
    > - All statements in this application are true and correct to the best of your knowledge and belief
    > 
    > - Since the applicant is an active-duty service member, he or she has consulted with an Education Service Officer (ESO) regarding their education program

    c. If the applicant is over 18 years of age and answered "Have you ever served on active duty in the armed services?" with "No":

    > **By submitting this form** you certify that all statements in this application are true and correct to the best of your knowledge and belief.

    d. If the applicant is over 18 years of age and answered "Have you ever served on active duty in the armed services?" with "Yes" and and **_any_** service period "Service end date" is blank:

    > **By submitting this form** you certify that:
    > 
    > - All statements in this application are true and correct to the best of your knowledge and belief
    > 
    > - As an active-duty service member, you have consulted with an Education Service Officer (ESO) regarding your education program

    e. If the applicant answered the "Your date of birth" with a birthdate that shows that they are under 18 years of age and answered "Have you ever served on active duty in the armed services?" with "Yes" and all service period "Service end date" values are **_not_** blank:

    > **By submitting this form** you certify that:
    > 
    > - You are the parent, guardian, or custodian of the applicant
    > 
    > - All statements in this application are true and correct to the best of your knowledge and belief

    f. If the applicant is over 18 years of age and answered "Have you ever served on active duty in the armed services?" with "Yes" and all service period "Service end date" values are **_not_** blank:

    > **By submitting this form** you certify that all statements in this application are true and correct to the best of your knowledge and belief.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/5490/helpers.jsx [hunk 1]
```diff
@@ -84,3 +102,30 @@ export const benefitsDisclaimerChild = (
...
+      </p>
+    </div>
+  </div>
+);
+
+export const eighteenOrOver = birthday => {
+  return (
+    birthday === undefined ||
+    moment().diff(moment(birthday, 'YYYY-MM-DD'), 'years') > 17 ||
+    birthday.length !== 10
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/5490/helpers.jsx:129]: Should probably check valid date length prior to checking years

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/5490/helpers.jsx (hunk 1 above)
Comment (elder-joseph) [reply src/applications/edu-benefits/5490/helpers.jsx:129]: Probably a good idea there. I will switch them around.

File: src/applications/edu-benefits/5490/components/applicantInformationUpdate.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * Returns an applicantInformation page based on the options passed.
+ *
+ * @param {Object} schema   The full schema for the form
+ * @param {Object} options  Options to override the defaults above
+ */
+export default function applicantInformationUpdate(schema, options) {
+  // Use the defaults as necessary, but override with the options given
+  const prefix = options && options.isVeteran ? 'veteran' : 'relative';
+  const mergedOptions = Object.assign({}, defaults(prefix), options);
+  const { fields, required, labels } = mergedOptions;
```
[the remark this answers is not in the document]
Comment (elder-joseph) [reply src/applications/edu-benefits/5490/components/applicantInformationUpdate.jsx:37]: I don't see why we couldn't do that. I just copied this section over from the platform `ApplicantInformation`, but will remove merged options here.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**